test/e2e/install/install_test.go

```diff
@@ -128,11 +129,15 @@ func installMayastor() {
 	createPools(&e2eCfg)
 
 	// Wait for pools to be online
-	Eventually(func() error {
-		return common.CheckAllPoolsAreOnline()
-	}, 120, 10,
-	).Should(BeNil())
-
+	const timoSecs = 120
+	const timoSleepSecs = 10
+	for ix := 0; ix < timoSecs/timoSleepSecs; ix++ {
+		time.Sleep(timoSleepSecs * time.Second)
+		err = common.CheckAllPoolsAreOnline(); if err == nil {
+			break
+		}
+	}
+	Expect(err).To(BeNil(), "All pools are not online")
```

Comment:
("first-glance" clarity of message): perhaps "Not all pools are online", or "One or more pools are offline", rather than "all are not online"?